mods/action.js

```diff
@@ -162,7 +165,8 @@ ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
 ActionService.prototype.query = function(restbase, req) {
     return this._doRequest(restbase, req, {
         action: 'query',
-        format: 'json'
+        format: 'json',
+        rawcontinue: ''
```

Comment:
As per https://lists.wikimedia.org/pipermail/wikitech-l/2015-June/081931.html it should be

```js
rawcontinue: 1
```